code/scripting/api/objs/oswpt.cpp

```diff
@@ -0,0 +1,183 @@
+//
+//
+
+#include "oswpt.h"
+#include "parse_object.h"
+#include "ship.h"
+#include "team.h"
+#include "waypoint.h"
+#include "wing.h"
+#include "ship/ship.h"
+
+namespace scripting {
+namespace api {
+
+//**********HANDLE: Wing
+ADE_OBJ(l_OSWPT, object_ship_wing_point_team, "oswpt", "OSWPT handle");
```

Comment:
`OSWPT` isn't really useful in this context since it does not help in explaining what this handle does. Please add a better description here which explains that this is a combined handle to either of those categories.